backup_and_restore/application_backup_and_restore/installing/installing-oadp-ocs.adoc

```diff
@@ -21,7 +21,7 @@ You create a `Secret` for the backup location and then you install the Data Prot
 
```

Comment:
Is the [link at line 15](installing-oadp-ocs) right? It links to the MCG assembly.